### PR TITLE
Update microsoft-teams-user-activity-preview.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to total meetings decription to reduce ambiguity

Fixeshttps://github.com/MicrosoftDocs/microsoft-365-docs/issues/12540

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/activity-reports/microsoft-teams-user-activity-preview.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ To ensure data quality, we perform daily data validation checks for the past thr
 |Replies   <br/> |The number of replied messages in all channels during the specified time period. <br/> |
 |Urgent messages    <br/> |The number of urgent messages during the specified time period. <br/> |
 |Chat messages   <br/> |The number of unique messages that the user posted in a private chat during the specified time period.  <br/> |
-|Total meetings   <br/> |The number of online meetings that the user participated in during the specified time period.  <br/> |
+|Total meetings   <br/> |The sum of online meetings that the user Organized and participated in during the specified time period.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/microsoft-teams-user-activity-preview.md]:
```suggestion
|Total meetings   <br/> |The sum of online meetings that the user organized and participated in during the specified time period.  <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@camillepack Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@camillepack Gentle reminder

**Pull request merged**